resources/js/app/components/drop-upload.js

```diff
@@ -196,6 +209,28 @@ export default {
             this.$forceUpdate();
         },
 
+        uploadSuccessful(file, object) {
+            this.$emit('new-relations', [object]);
+            this.removeProgressItem(file);
+        },
+
+        getBaseNameFile(file) {
```

Comment:
this function could be in `resources/js/app/helpers/view.js` with other "files" functions, as `checkMaxFileSize`; in this way it could be reused by `this.$helpers.getBaseNameFile(file)`